### DISCUSSION TITLE
Social media Icon

### DISCUSSION THREAD

**Discussion by Codehouze** [tailwindlabs/heroicons#896] [Ideas]
Add Social Media Icons like LinkedIn, Twitter, Youtube, GitHub,  and others. thanks

**Comment by agboolaidris** [comment]
@RobinMalfait Can I go ahead to work on this?

**Comment by zefir-git** [comment]
Why? And what about social networks like Mastodon? Perhaps after that Discord, Signal, Matrix, Element, WhatsApp, Viber and all other random chat apps. After social media let's add the fortune 500 brands because surely people will want Microsoft, Google, GitHub, etc. Then since we now have thousands of icons already, why not start adding any random company?

Brand icons are light years outside of the scope of Heroicons IMO. And not to mention, there already is a free open-source project that actively maintains thousands of brand icons: https://simpleicons.org/
